src/components/Swap.tsx

```diff
@@ -825,17 +826,58 @@ const Swap = () => {
         </Button>
       )
     }
+    const fromAmountUSD = new BigNumber(routes[highlightedIndex]?.fromAmountUSD)
+    const toAmountUSD = new BigNumber(routes[highlightedIndex]?.toAmountUSD)
+    const gasCostUSD = new BigNumber(routes[highlightedIndex]?.gasCostUSD || 0)
+    const allValuesAvailable =
+      !fromAmountUSD.isZero() && !toAmountUSD.isZero() && !gasCostUSD.isZero()
 
-    return (
-      <Button
-        disabled={highlightedIndex === -1}
-        shape="round"
-        type="primary"
-        icon={<SwapOutlined />}
-        size={'large'}
-        onClick={() => openModal()}>
-        Swap
-      </Button>
+    const totalExpenditure = fromAmountUSD.plus(gasCostUSD)
+    const amountReceivedPercentage = toAmountUSD.dividedBy(totalExpenditure)
+    const receivedAmountTooLow = amountReceivedPercentage.isLessThan(new BigNumber(0.75))
```

Comment:
We might want to be able to disable this thing here for our development I think 😅 I guess most of our test transfers might not be suitable here and we don't want to hide that message every time?

Reply:
Maybe something like a "don't show again" flag in local storage?